docs/testing.md

```diff
@@ -0,0 +1,55 @@
+# Testing Storefront
+
+This document lists the features and flows supported in Storefront. This is not an exhaustive list.
+
+## WooCommerce core
+- [ ] single product page
+- [ ] shop page / product catalog / product list
+- [ ] search for products
+- [ ] sale products
+- [ ] add to cart, view cart, manage cart contents
+- [ ] use coupon
+- [ ] checkout
+- [ ] log in as customer, view account info
+- [ ] customer can view previous orders
+- [ ] visitor/customer can review products
+- [ ] visitors can view reviews
+- [ ] woo widgets
```

Comment:
Ideally I think most bullet points should have at least one verb that specifies what needs to be tested. In some cases it might be a bit ambiguous otherwise (do I need to check whether I can add Woo Widgets, whether they appear or whether there have style regressions?).

Reply:
I totally agree – and this helps to soften the checklist and focus on the real need, rather than how it's implemented. I'll do another pass on this.